CodePush.js

```diff
@@ -330,4 +294,4 @@ var CodePush = {
   }
 };
 
-module.exports = CodePush;
+export default CodePush;
```

Comment:
We reference CodePush a few times in this file, e.g. `syncStatusChangeCallback(CodePush.SyncStatus...` prefer to keep this here for simplicity